app/addons/activetasks/actions.js

```diff
@@ -16,17 +16,18 @@ define([
 ],
 function (FauxtonAPI, ActionTypes, Resources) {
   return {
-    fetchAndSetActiveTasks: function (options) {
+    init: function (options) {
       var activeTasks = options;
```

Comment:
You could remove this line if you changed the function declaration to `init: function (activeTasks) { `